scripts/seeder/main.go

```diff
@@ -48,8 +52,12 @@ func main() {
 		seedProductType(ctx)
 	case "uom":
 		seedUOM(ctx)
+	case "vendor":
+		seedvendor(ctx)
+	case "product-vendor":
+		seedProductVendor(ctx)
 	default:
-		log.Println("Usage: go run scripts/seeder/main.go [product|product_category|product_type|uom].")
+		log.Println("Usage: go run scripts/seeder/main.go [product|product_category|product_type|uom|vendor|product-vendor].")
```

Comment:
i think we can have a convention either using underscore or dash, I don't really have a preference to any, can you adjust them?